### PR TITLE
Added invalid market rules for disputing and reporting

### LINKED ISSUE #4519: Reporting and Disputing: Add rules section to top of form
Add the rules section into the top of the reporting and disputing forms. Use the same design across both.

Design
https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=3242%3A109036

****Use the Same Rules as Market Creation

### PR DESCRIPTION
**Pull request by bconfortin**
Closes #4519

### CODE REVIEW

File: packages/augur-ui/src/modules/reporting/common.styles.less [hunk 1]
```diff
@@ -589,6 +589,10 @@ div.ReportingSubheaders {
     .text-16-medium;
 
     color: @color-primary-text;
+
+    > a {
+      word-break: break-word;
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/reporting/common.styles.less:594]: Fixes reporting cards for lower resolutions (iPhone 11 Pro/iPhone 11 Pro Max).

Markets with lots of texts were overflowing to the right, making the report button unclickable.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 💯
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/654hj30**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/reporting/common.styles.less (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/reporting/common.styles.less:594]: Lgtm

**Review by bthaile — APPROVED** [review]
Lgtm

**Pull request merged**